src/pymor/vectorarrays/list.py

```diff
@@ -353,9 +353,11 @@ class ListVectorArray(VectorArray):
     The associated |VectorSpace| is a subclass of
     :class:`ListVectorSpace`.
 
-    For an example, see :class:`NumpyVector`, :class:`NumpyListVectorSpace`
-    or :class:`~pymor.bindings.fenics.FenicsVector`,
-    :class:`~pymor.bindings.fenics.FenicsVectorSpace`.
+    For an example, see :class:`NumpyVector` and :class:`NumpyListVectorSpace`,
+    :class:`~pymor.bindings.fenics.FenicsVector` and
+    :class:`~pymor.bindings.fenics.FenicsVectorSpace`,
+    :class:`~pymor.bindings.dunegdt.DuneXTVector` and
+    :class:`~pymor.bindings.dunegdt.DuneXTVectorSpace`.
```

Comment:
I'm not sure if we need to list all implementors here. If we do, we should also at least mention NGSolve ..

Reply:
I felt that its useful for those adding 3rd party PDE solvers to have as many examples as possible to look at. So for the moment I added ngsolve in 3ecee16ef4332adbd68f5518a758baec05c1033e, but we can also drop it if it gets too much!